packages/core/src/tests/modal/modal.e2e.ts

```diff
@@ -85,16 +89,17 @@ test('modal with dropdown', async ({ mount, page }) => {
       });
 
       await mount(`
-      <ix-modal size="${size}">
-        <ix-modal-header>Header</ix-modal-header>
-        <ix-modal-content>Some Content 123 content 123 content 123 content 123</ix-modal-content>
-        <ix-modal-footer>
-          <ix-button>Test</ix-button>
-        </ix-modal-footer>
-      </ix-modal>
-    `);
+        <ix-modal size="${size}">
+          <ix-modal-header>Header</ix-modal-header>
+          <ix-modal-content>Some Content 123 content 123 content 123 content 123</ix-modal-content>
+          <ix-modal-footer>
+            <ix-button>Test</ix-button>
+          </ix-modal-footer>
+        </ix-modal>
+      `);
 
       const modal = page.locator('ix-modal');
+      await modal.evaluate((modal) => (modal as any).showModal());
```

Comment:
Why add another showModal()?

Reply:
Because thats the way how the modals work.